frontend/packages/data-portal/app/components/Deposition/MethodLinks/MethodLinksMetadataTable.tsx

```diff
@@ -0,0 +1,226 @@
+import { useMemo } from 'react'
+
+import { Accordion } from 'app/components/Accordion'
+import { Link } from 'app/components/Link'
+import { MetadataTable } from 'app/components/Table'
+import { methodLabels, MethodType } from 'app/constants/methodTypes'
+import { useI18n } from 'app/hooks/useI18n'
+import { getTableData } from 'app/utils/table'
+
+import { generateMethodLinks, MethodLinkVariantProps } from './common'
+
+const COLUMN_WIDTH = 170
+
+type MethodData = {
```

Comment:
Some of our types end with `Type` (like `MethodType` below), but others don't.  This would be nice to standardize as well.